pkg/minishift/registration/utils.go

```diff
@@ -60,6 +66,10 @@ func RegisterHostVM(host *host.Host, param *RegistrationParameters) error {
 
 // Unregister host VM
 func UnregisterHostVM(host *host.Host, param *RegistrationParameters) error {
+	if param.SkipUnRegistration {
+		log.Debug("User choosed to skip un-registration")
```

Comment:
The caveat here is that , if user deletes the VM i.e `minishift delete` the entitlement will not be freed. So we should mention this to user IMO.